### PR TITLE
CMT 4K 2017: Update match schedule, mappool and results.

### PR DESCRIPTION
**Pull request by lzc731524299**

### CODE REVIEW

**Comment by kj415j45** [comment]
把标题格式统一一下:
>CMT 4K 2017: XXXXXXXX

**Comment by Loctav** [comment]
Why are the tables screenshots instead of like, you know, actual tables?

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 1]
```diff
@@ -51,20 +51,22 @@ Update after group stage starts.
 
 ## Match Preview
 
-### Qualification Stage
+### Group Stage
 
-Saturday, 24. June 2017 15:00-20:00 (UTC+8)
-![6/24](https://user-images.githubusercontent.com/18349191/27418509-2baea9b6-574e-11e7-988b-99b4bcd90db4.png)
+Friday, 30 June 2017
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: Add dot after 30

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 2]
```diff
@@ -51,20 +51,22 @@ Update after group stage starts.
...
+### Group Stage
 
-Saturday, 24. June 2017 15:00-20:00 (UTC+8)
-![6/24](https://user-images.githubusercontent.com/18349191/27418509-2baea9b6-574e-11e7-988b-99b4bcd90db4.png)
+Friday, 30 June 2017  
+![30/6](https://user-images.githubusercontent.com/28429414/27581330-6a722f3c-5afb-11e7-98c9-a975262bad35.png)
 
-Sunday, 25. June 2017 11:00-16:00 (UTC+8)
-![6/25](https://user-images.githubusercontent.com/18349191/27418526-478da5ba-574e-11e7-91ad-69416fa0d2fd.png)
+Saturday, 1 July 2017
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: add dot after 1

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 3]
```diff
@@ -51,20 +51,22 @@ Update after group stage starts.
...
-![6/24](https://user-images.githubusercontent.com/18349191/27418509-2baea9b6-574e-11e7-988b-99b4bcd90db4.png)
+Friday, 30 June 2017  
+![30/6](https://user-images.githubusercontent.com/28429414/27581330-6a722f3c-5afb-11e7-98c9-a975262bad35.png)
 
-Sunday, 25. June 2017 11:00-16:00 (UTC+8)
-![6/25](https://user-images.githubusercontent.com/18349191/27418526-478da5ba-574e-11e7-91ad-69416fa0d2fd.png)
+Saturday, 1 July 2017  
+![1/7](https://user-images.githubusercontent.com/28429414/27581331-6ce9b758-5afb-11e7-8128-226259848e90.png)
 
+Sunday, 2 July 2017
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: add dot after 2

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 4]
```diff
@@ -92,6 +94,25 @@ TB:
 [Kuroneko Dungeon - Ryuu to Shoujo to Decoherence \[Another\]](https://osu.ppy.sh/b/1319304)  
 [Ichigaki's 4K Collection Vol.1 \[TIEFSEE \[Marathon\]\]](https://osu.ppy.sh/b/845983)
 
+### Group Stage mappool
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: make mappools with bullet lists, same for other mappools

**Pull request merged**